core/services/synchronization/explorer_client.go

```diff
@@ -31,6 +31,8 @@ const (
 	ConnectionStatusConnected = ConnectionStatus("connected")
 	// ConnectionStatusError is used when there is an error
 	ConnectionStatusError = ConnectionStatus("error")
+	// SendBufferSize is the number of messages to keep in the buffer before dropping additional ones
+	SendBufferSize = 3
```

Comment:
I'd suggest making this larger. The memory requirements should be moderate, and we'd hopefully achieve a better delivery rate in case of brief connection issues.
```suggestion
	SendBufferSize = 100
```

Reply:
I'd also move this out of this `const` block which only contains ConnectionStatus values.